common/config/rush/pnpm-lock.yaml

```diff
@@ -113,7 +113,7 @@ packages:
     dependencies:
       '@azure-rest/core-client': 1.0.0-beta.6
       '@azure/core-paging': 1.1.3
-      '@azure/core-rest-pipeline': 1.1.1
+      '@azure/core-rest-pipeline': 1.2.0
```

Comment:
Interesting... why didnt the automated PRs include this change? #16857?

Reply:
because it runs `rush update --recheck` which minimizes the churn. In this PR, I ran `rush update --full` to shake off the dep on nodeauth which recheck would not be able to do.